ext/web/hr_timer_lock.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+#[cfg(target_os = "windows")]
+mod windows {
+  use std::marker::PhantomData;
+  use std::sync::atomic::AtomicU32;
+
+  pub(crate) struct HrTimerLock {
+    pub(super) _unconstructable: PhantomData<()>,
+  }
+
+  /// Decrease the reference count of the HR timer on drop.
+  impl Drop for HrTimerLock {
+    fn drop(&mut self) {
+      dec_ref();
+    }
+  }
+
+  /// Maintains the HR timer refcount. Times should not be nested more than 5 deep so this is
+  /// more than sufficient.
```

Comment:
What does "times should not be nested more than 5 deep" mean?

Reply:
Oops... typo. Should be timer. And re-reading the spec, I'm wrong -- timers can be arbitrarily nested, but there should not be an arbitrary # of simultaneous timers.